data-reconciliation-app/Makefile

```diff
@@ -18,10 +19,16 @@ build-enclave: build ## 📦 Build Enclave container image from Dockerfile
 	@echo -e "\e[34m$@\e[0m" || true
 	@../scripts/build_image.sh enclave
 
-# Start hosting the application using `sandbox.sh` 
-start-host: build ## 🏃 Start the CCF network using Sandbox.sh
+# Start hosting the application using `sandbox.sh` and enable custom JWT authentication
+start-host: build  ## 🏃 Start the CCF network using Sandbox.sh
 	@echo -e "\e[34m$@\e[0m" || true
-	@/opt/ccf_virtual/bin/sandbox.sh --js-app-bundle ./dist/ --initial-member-count 3 --initial-user-count 1 --constitution-dir ./governance/constitution
+	@/opt/ccf_virtual/bin/sandbox.sh --js-app-bundle ./dist/ --initial-member-count 3 --initial-user-count 1 --constitution-dir ./governance/constitution --jwt-issuer "./workspace/proposals/set_jwt_issuer_test_sandbox.json"
+
+# Start hosting the application using `sandbox.sh` and enable MS IDP JWT authentication
+start-host-jwt-ms: build ## 🏃 Start the CCF network using Sandbox.sh
+	@echo -e "\e[34m$@\e[0m" || true
+	@/opt/ccf_virtual/bin/sandbox.sh --js-app-bundle ./dist/ --initial-member-count 3 --initial-user-count 1 --constitution-dir ./governance/constitution --jwt-issuer "./workspace/proposals/set_jwt_issuer_ms_sandbox.json"
+
 
 test: build ## 🧪 Test the Data Reconciliation Application in the sandbox
```

Comment:
Is it possible to add test cases to `make test` to do JWT authentication? I heard that JWT authentication is added for swagger, but I think automated tests are necessary to prevent degradation.

Reply:
I've created a new issue to capture it #191